bindings/python/setup.py

```diff
@@ -22,10 +22,10 @@ def get_py_version_cfgs():
 
 setup(
     name="iota_wallet",
-    version="0.2.0",
+    version="1.0.0.rc.1",
     classifiers=[
         "License :: SPDX-License-Identifier ::  Apache-2.0",
-        "Development Status :: 0.1.0 - Alpha",
+        "Development Status :: 1.0.0.rc.1",
```

Comment:
```suggestion
        "Development Status :: 1.0.0-rc.1",
```
?

Reply:
Or maybe python doesn't like that ?

Reply:
Makes sense, will also update in iota.rs